chromium_src/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc

```diff
@@ -253,6 +253,7 @@ void BraveAddCommonStrings(content::WebUIDataSource* html_source,
     { "braveNewTabBraveRewards", IDS_SETTINGS_NEW_TAB_BRAVE_REWARDS },
     { "braveNewTabBinance", IDS_SETTINGS_NEW_TAB_BINANCE },
     { "braveNewTabGemini", IDS_SETTINGS_NEW_TAB_GEMINI },
+    { "braveNewTabBitcoinDotCom", IDS_SETTINGS_NEW_TAB_BITCOIN_DOT_COM },
```

Comment:
Should be within `#if BUILDFLAG(BITCOIN_DOT_COM_ENABLED)`

Reply:
fixed